packages/modules/viewer-react/src/components/app-ui/widgets/BasicNavigationWidget.tsx

```diff
@@ -2,9 +2,14 @@
  * Copyright (c) Bentley Systems, Incorporated. All rights reserved.
  * See LICENSE.md in the project root for license terms and full copyright notice.
  *--------------------------------------------------------------------------------------------*/
+/*---------------------------------------------------------------------------------------------
+ * Copyright (c) Bentley Systems, Incorporated. All rights reserved.
+ * See LICENSE.md in the project root for license terms and full copyright notice.
```

Comment:
looks like the precommit script added a duplicate header. Can you remove one of them?